[2024-05-05T10:22:35.728+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-05T10:22:35.809+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_complex.create_entry_gcs manual__2024-05-05T10:22:28.612296+00:00 [queued]>
[2024-05-05T10:22:35.825+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_complex.create_entry_gcs manual__2024-05-05T10:22:28.612296+00:00 [queued]>
[2024-05-05T10:22:35.828+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-05T10:22:35.868+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): create_entry_gcs> on 2024-05-05 10:22:28.612296+00:00
[2024-05-05T10:22:35.891+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1975) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-05T10:22:35.904+0000] {standard_task_runner.py:63} INFO - Started process 2004 to run task
[2024-05-05T10:22:35.905+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_complex', 'create_entry_gcs', 'manual__2024-05-05T10:22:28.612296+00:00', '--job-id', '5', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_complex.py', '--cfg-path', '/tmp/tmpxopspi1y']
[2024-05-05T10:22:35.910+0000] {standard_task_runner.py:91} INFO - Job 5: Subtask create_entry_gcs
[2024-05-05T10:22:35.971+0000] {task_command.py:426} INFO - Running <TaskInstance: example_complex.create_entry_gcs manual__2024-05-05T10:22:28.612296+00:00 [running]> on host b2b5d11d7cf5
[2024-05-05T10:22:36.070+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_complex' AIRFLOW_CTX_TASK_ID='create_entry_gcs' AIRFLOW_CTX_EXECUTION_DATE='2024-05-05T10:22:28.612296+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-05T10:22:28.612296+00:00'
[2024-05-05T10:22:36.071+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-05T10:22:36.072+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-05T10:22:36.076+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo create_entry_gcs']
[2024-05-05T10:22:36.095+0000] {subprocess.py:86} INFO - Output:
[2024-05-05T10:22:36.099+0000] {subprocess.py:93} INFO - create_entry_gcs
[2024-05-05T10:22:36.099+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-05T10:22:36.100+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-05T10:22:36.133+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=example_complex, task_id=create_entry_gcs, execution_date=20240505T102228, start_date=20240505T102235, end_date=20240505T102236
[2024-05-05T10:22:36.169+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-05T10:22:36.207+0000] {taskinstance.py:3482} INFO - 6 downstream tasks scheduled from follow-on schedule check
[2024-05-05T10:22:36.217+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
